doc/nvim-tree-lua.txt

```diff
@@ -1219,7 +1220,8 @@ In the event of a problem please disable the experiment and raise an issue.
 Configuration for diagnostic logging.
 
     *nvim-tree.log.enable*
-    Enable logging to a file `$XDG_CACHE_HOME/nvim/nvim-tree.log`
+    Enable logging to a file `nvim-tree.log` in |stdpath| `"cache"`, usually
```

Comment:
Will follow up with change to `log` location instead of `cache` for logs.

Reply:
Yes! cache is a bit odd... there must be a better place...

Reply:
#2162